docker-test/do_bsm_test.sh

```diff
@@ -15,14 +26,34 @@ echo "**************************"
 echo "Consuming Filtered BSMs at offset "$offset "..." 
 echo "**************************"
 
+attempts=0
+max_attempts=5
 while true; do
-    /cvdi-stream-build/kafka-test/kafka_tool -C -b $broker -p 0 -t topic.FilteredOdeBsmJson -e -o $offset 2> con.err | /cvdi-stream/docker-test/test_out.py > tmp.out
+    attempts=$((attempts+1))
+
+    timeout 5 /cvdi-stream-build/kafka-test/kafka_tool -C -b $broker -p 0 -t topic.FilteredOdeBsmJson -e -o $offset 2> con.err | /cvdi-stream/docker-test/test_out.py > tmp.out
```

Comment:
![image](https://github.com/usdot-jpo-ode/jpo-cvdp/assets/62157949/275f7134-0675-4dd3-8700-4d5af3d372f7)
The pipe break and it is not sending data to Kafka.

Reply:
The `do_bsm_test.sh` script isn't intended to be run directly. I believe it is used by the `standalone.sh` and `standalone_multi.sh` scripts, which are in turn used by the `do_kafka_test.sh` script. Do you experience this behavior when running the `do_kafka_test.sh` script?

Reply:
I am not able to finish running the do_kafka_test.sh as some data are missing below (or I do not have those data in my ubuntu): 
```
/ppm_data/road_file.csv
/tmp/docker-test/data
```